src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -598,7 +598,7 @@ void InstanceKlass::deallocate_contents(ClassLoaderData* loader_data) {
   // Can't release the constant pool here because the constant pool can be
   // deallocated separately from the InstanceKlass for default methods and
   // redefine classes.
```

Comment:
Does this comment make sense when the parameter doesn't actually control release of the CP?

Reply:
It should be inclusive of MethodData also.